### ISSUE TITLE
Visibility of primary navigation on landing page

### ISSUE THREAD

**Issue by FranzSkuffka** [carrot/roots-www#31]
I wonder why there are no navigation options in the top right corner of the landing page.
To me this seems to be causing inner (vs. sub-pages) and outer inconsistency (vs. other websites).

In my eyes, a sidebar which is expanded by default yielding collapsed sub-entries would be the best solution for a site like this. This is based on the assumption that a major part of the visitors of roots.cx are looking for the content that is, right now, mentioned at the bottom of the landing page.

**Comment by hhsnopek** [comment]
PR's are always welcome :smile:

**Comment by jenius** [comment]
Yep agreed. Could have been designer better, this is my fault haha
